tox.ini

```diff
@@ -23,6 +23,7 @@ changedir = {toxinidir}/cfgov
 commands =
     coverage erase
     coverage run --source='.' manage.py test {posargs}
+    flake8
```

Comment:
That is a very good idea! I generally use tox the same way, and that hadn't occurred to me.